### PR TITLE
Added Theme aqua

### PR DESCRIPTION
**Pull request by IronManCool001**
- [√ ] New feature (non-breaking change which adds functionality)
- [✓ ] This change requires a documentation update

Added aqua theme please check the src/features/Colors_and_Theme/color.css

Link to update: [aqua theme](https://github.com/IronManCool001/fire-ui/blob/master/src/feature/Colors_and_Theme/color.css)

### CODE REVIEW

File: src/feature/Colors_and_Theme/color.css [hunk 1]
```diff
@@ -7,7 +7,11 @@
   background-color: #eef0f2 !important;
   color: #000 !important;
   transition: 0.5s; }
-
+[data-theme="aqua"], .aqua-mode {
+  background-color: #0070f3 !important;
+  color: #000000 !important
```
Comment (kimlimjustin) [inline src/feature/Colors_and_Theme/color.css:12]: You should add a semicolon here on line 12 to make the CSS work.

File: src/feature/Colors_and_Theme/color.css [hunk 2]
```diff
@@ -25,10 +29,10 @@
 
 .green-text {
   color: #33ff0a !important; }
-
+.aqua-text{color:#000;}
```
Comment (kimlimjustin) [inline src/feature/Colors_and_Theme/color.css:32]: Aqua-text value is black color? I think you misunderstood this, for the `color`-text value is the color itself, you can view the prev docs [here](https://fire-ui.netlify.app/docs/colors#colors)

**Review by kimlimjustin — COMMENTED** [review]
In addition, I'll make additional commits into this PR before merging it.

File: src/feature/Colors_and_Theme/color.css (hunk 1 above)
Comment (IronManCool001) [reply src/feature/Colors_and_Theme/color.css:12]: Yeah I forgot to add that

Comment (kimlimjustin) [reply src/feature/Colors_and_Theme/color.css:12]: Ill add it, Im editing some files rn regarding this PR

**Comment by kimlimjustin** [comment]
I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?

File: src/feature/Colors_and_Theme/color.css (hunk 2 above)
Comment (IronManCool001) [reply src/feature/Colors_and_Theme/color.css]: I set it to **black** cause it thought it the contrast would be ok but if you want to change the text color you can do it

Comment (kimlimjustin) [reply src/feature/Colors_and_Theme/color.css]: Ohh, I made it to the origin color to make the user chance to pick other colors, not only a certain color. Btw, I've changed that, for more info, visit this comment https://github.com/fire-ui/fire-ui/pull/39#issuecomment-755973856

**Comment by IronManCool001** [comment]
> I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?

Make switch theme so that it applies to all themes

**Comment by kimlimjustin** [comment]
> > I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?
> 
> Make switch theme so that it applies to all themes

So, switch theme will switch from aqua => light => dark => aqua => light => dark, and so on?

**Comment by IronManCool001** [comment]
> > > I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?
> > 
> > 
> > Make switch theme so that it applies to all themes
> 
> So, switch theme will switch from aqua => light => dark => aqua => light => dark, and so on?

Yeah like that only

**Comment by stanleyowen** [comment]
> > > > I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?
> > > 
> > > 
> > > Make switch theme so that it applies to all themes
> > 
> > 
> > So, switch theme will switch from aqua => light => dark => aqua => light => dark, and so on?
> 
> Yeah like that only

Hi @IronManCool001, I think it'll be quite weird for a website to use aqua => light => dark method (cause the major website I usually visited is using only `light` and `dark` mode). So how about let's change `switch-theme` become `data-switch-theme` with the value of the target theme to switch?

**Comment by kimlimjustin** [comment]
> > > > > I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?
> > > > 
> > > > 
> > > > Make switch theme so that it applies to all themes
> > > 
> > > 
> > > So, switch theme will switch from aqua => light => dark => aqua => light => dark, and so on?
> > 
> > 
> > Yeah like that only
> 
> Hi @IronManCool001, I think it'll be quite weird for a website to use aqua => light => dark method (cause the major website I usually visited is using only `light` and `dark` mode). So how about let's change `switch-theme` become `data-switch-theme` with the value of the target theme to switch?

I think @stanleyowen  idea is good. I'll implement his idea.

**Comment by kimlimjustin** [comment]
This PR will be merged soon after documented.

**Comment by kimlimjustin** [comment]
Fire UI theme is OK rn, you are welcomed to add more theme into Fire UI.

**Comment by IronManCool001** [comment]
> > > > > > I've added a commit and now `[data-theme = "aqua"]`, `aqua-text`, `aqua-background`, `[data-theme="aqua"] .theme-reverse` and `[data-theme="aqua"] .theme-adjust` classes work. However, I'm still thinking of the `switch-theme` attribute of how it's supposed to work (either `switch-theme` attribute only for light and dark theme or `switch-theme` attribute will switch all themes). Any thoughts?
> > > > > 
> > > > > 
> > > > > Make switch theme so that it applies to all themes
> > > > 
> > > > 
> > > > So, switch theme will switch from aqua => light => dark => aqua => light => dark, and so on?
> > > 
> > > 
> > > Yeah like that only
> > 
> > 
> > Hi @IronManCool001, I think it'll be quite weird for a website to use aqua => light => dark method (cause the major website I usually visited is using only `light` and `dark` mode). So how about let's change `switch-theme` become `data-switch-theme` with the value of the target theme to switch?
> 
> I think @stanleyowen idea is good. I'll implement his idea.

It is better that we use a pop-up which allows the user to select which theme he/she wants to use.

**Comment by kimlimjustin** [comment]
Hmm, I think we can make it as an additional feature alongside the current feature. Do you want to be assigned? @IronManCool001

**Comment by IronManCool001** [comment]
> Hmm, I think we can make it as an additional feature alongside the current feature. Do you want to be assigned? @IronManCool001

Currently I am busy so won't be able to help you but for any help contact me using technicalfriend.help@gmail.com

**Comment by kimlimjustin** [comment]
Okay, I'll implement it as soon as possible.

**Pull request merged**